_posts/2023-12-22-central-workflows.markdown

```diff
@@ -0,0 +1,197 @@
+---
+layout: post
+title:  "DRY in GitHub Actions with reusable workflows"
```

Comment:
`"DRY in GitHub Actions with Reusable Workflows"`

I'd capitalize the entire title, looks better